flow/project.py

```diff
@@ -3435,28 +3385,16 @@ def _main_run(self, args):
 
     def _main_script(self, args):
         "Generate a script for the execution of operations."
-        if args.requires and not args.cmd:
-            raise ValueError(
-                "The --requires option can only be used in combination with --cmd.")
-        if args.cmd and args.operation_name:
-            raise ValueError(
-                "Cannot use the -o/--operation-name and the --cmd options in combination!")
         # Select jobs:
         jobs = self._select_jobs_from_args(args)
 
         # Gather all pending operations or generate them based on a direct command...
         with self._potentially_buffered():
-            if args.cmd:
-                warnings.warn("The --cmd option for script is deprecated as of "
-                              "0.9 and will be removed in 0.11.",
-                              DeprecationWarning)
-                operations = self._generate_operations(args.cmd, jobs, args.requires)
```

Comment:
We should delete `FlowProject._generate_operations` as it is used no where else.

Reply:
@b-butler I have deleted this method. I think we can go ahead.

Reply:
Thanks for pointing that out. I missed it in my review.